relay-profiling/src/types.rs

```diff
@@ -0,0 +1,14 @@
+use serde::{Deserialize, Serialize};
+
+use crate::native_debug_image::NativeDebugImage;
+
+#[derive(Default, Debug, Serialize, Deserialize, Clone)]
+pub struct DebugMeta {
```

Comment:
Can you document these types?

Reply:
These are the same fields as in Relay but serde-friendly. I added a comment to explain that.